docs/experiment/experiment.rst

```diff
@@ -0,0 +1,81 @@
+Experiment
+==========
+
+The main component of an experiment in :py:mod:`EasyReflectometry` is the `model`. 
+This is a description of the `sample` and the environment in which the experiment is performed. 
+The `Model` is used to calculate the reflectivity of the `sample` at a given set of angles (`q` values).
+The `resolution_functions` are used to account for non-ideal effects in the instrument.
```

Comment:
I would reword "The `resolution_functions` are used to account for non-ideal effects in the instrument." to
"The `resolution_functions` are used to quantify the experimental uncertainties in wavelength and angle, allowing the `Model` to accurately describe the data."

Reply:
done